src/ec2/model.ts

```diff
@@ -90,11 +92,17 @@ export class Ec2ConnectionManager {
             )
         }
 
-        throw new ToolkitError('Is SSM running on the target instance?', {
+        if (!isSsmAgentRunning) {
+            throw new ToolkitError('Is SSM running on the target instance?', {
+                code: 'EC2SSMAgentStatus',
+                documentationUri: vscode.Uri.parse(
+                    'https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-getting-started.html'
```

Comment:
since this message is about the agent would this be a more relevant url:

https://docs.aws.amazon.com/systems-manager/latest/userguide/ssm-agent.html